lib/svrquery/protocol/sqp/register.go

```diff
@@ -5,5 +5,6 @@ import (
 )
 
 func init() {
-	protocol.MustRegister("sqp", newCreator)
+	protocol.MustRegister("sqp", newCreator(ServerInfo))
+	protocol.MustRegister("sqp2", newCreator(ServerInfo|Metrics))
```

Comment:
We should probably use the same version formatting as tf2e does here. So sqp-v2
https://github.com/Unity-Technologies/go-svrquery/blob/master/lib/svrquery/protocol/titanfall/register.go#L12